docs/documentation/sitespeed.io/configuration/index.md

```diff
@@ -147,10 +147,13 @@ s3
   --s3.key                The S3 key
   --s3.secret             The S3 secret
   --s3.bucketname         Name of the S3 bucket
-  --s3.path               Override the default folder path in the bucket where the results are uploaded. By default it's "DOMAIN_OR_FILENAME/TIMESTAMP", or the name of the
-                          folder if --outputFolder is specified.
+  --s3.path               Override the default folder path in the bucket where the results are uploaded. By default it's "DOMAIN_OR_FILENAME/TIMESTAMP", or the name of the folder if --outputFolder is specified.
   --s3.region             The S3 region. Optional depending on your settings.
-  --s3.removeLocalResult  Remove all the local result files after they have been uploaded to S3                                                     [boolean] [default: false]
+  --s3.endpoint           S3 endpoint. Defaults to s3.amazonaws.com
```

Comment:
Services such as [Minio](https://www.minio.io/) or [Ceph](http://docs.ceph.com/docs/master/radosgw/s3/) exposes the same API as S3 and are free or for internal usage. We need to be able to pass that endpoint to the underlying s3 library. In my use case, we want to store our reports in Ceph.